doc/tutorials/using_zephyr_as_uos.rst

```diff
@@ -0,0 +1,100 @@
+.. _using_zephyr_as_uos:
+
+Using Zephyr as User OS
+#######################
+
+The tutorial describes the steps to run zephyr as User OS on ACRN hypervisor. We are using
+Kaby Lake-based NUC model NUC7i5DNHE and you can try other platform ACRN supported as well.
+
+Introduction to Zephyr
+**********************
+
+The Zephyr is a scalable real-time operating-system (RTOS) supporting multiple hardware architecture,
+optimized for resource constrained devices, and built with safety and security in mind.
+
+Steps for Using Zephyr as User OS
+*********************************
+
+#. Build Zephyr
+
+   First of all, please follow the `Getting Started Guide
+   <https://docs.zephyrproject.org/latest/getting_started/>`_ to
+   setup Zephyr development environment. Then follow the `Board-Specfic Support
+   <https://docs.zephyrproject.org/latest/boards/index.html>`_ to build the acrn supported zephyr image ``zephyr.elf``.
+   
+   .. note::
```

Comment:
This note should not be included in the documentation.  We can't publish this doc until the Zephyr PR is merged.

Reply:
Got it.